### PR TITLE
Add version info to minikube-iso

### PR DESCRIPTION
**Pull request by r2d4**
Open to put it somewhere other than `/etc/version`
Checked in the generate file just for simplicity.

### CODE REVIEW

**Comment by r2d4** [comment]
Fixes #1168

File: Makefile [hunk 1]
```diff
@@ -84,6 +84,7 @@ localkube-image: out/localkube
 	make -C deploy/docker VERSION=$(VERSION)
 
 minikube_iso:
+	echo $(ISO_VERSION) > deploy/iso/minikube-iso/board/coreos/minikube/rootfs-overlay/etc/VERSION
```
Comment (dlorenc) [inline Makefile]: Would you mind .gitignoring it? This will mean we'll end up with a dirty tree state during a normal iso build.

Comment (r2d4) [inline Makefile]: ah yeah makes sense. fixed

**Comment by dlorenc** [comment]
We can add an iso_test to make sure this is present later.

**Pull request merged**